README.md

```diff
@@ -185,5 +185,10 @@ in-memory storage may require much RAM, for example, the Linux kernel takes over
 for the Linux kernel in "couples" mode is 1.5 GB and takes more than an hour / 180GB RAM to be
 parsed. However, most of the repositories are parsed within a minute.
 
+2. to use matplotlib in macOS and avoid errors us
```

Comment:
We should move it to "Plotting" section - there is already a way described with `--backend`, this is an alternative